napari/_qt/qt_viewer.py

```diff
@@ -165,8 +165,17 @@ def __init__(self, viewer: Viewer, show_welcome_screen: bool = False):
 
         # This dictionary holds the corresponding vispy visual for each layer
         self.layer_to_visual = {}
-        self.viewerButtons.consoleButton.clicked.connect(
-            self.toggle_console_visibility
+        action_manager.register_action(
+            "toggle_console_visibility",
```

Comment:
Using keyword args (as suggested by @jni) is a bit more verbose, but would make things easier to parse, read, understand.